MarvelHeroesApp/ClassExtension.swift

```diff
@@ -6,27 +6,26 @@
 //  Copyright © 2019 Lonely Tree Std. All rights reserved.
 //
 
-import UIKit
+import Foundation
 import CommonCrypto
 
 extension String {
+  func md5() -> String {
+    let messageData = self.data(using: .utf8)!
```

Comment:
You should avoid force unwrapping if it possible. Let's use guards or optional chaining instead of it.